control-spec.txt

```diff
@@ -1537,14 +1537,17 @@
     "HSFETCH" SP (HSAddress / "v" Version "-" DescId)
               *[SP "SERVER=" Server] CRLF
 
-    HSAddress = 16*Base32Character
+    HSAddress = 16*Base32Character (v2) or 56*Base32Character (v3)
```

Comment:
This won't pass the spec human linter that is atagar :).

I think the proper syntax is: `16*Base32Character / 56*Base32Character` and then below you would describe that for `Version=3`, the `HSAddress is the 56*...